src/main/java/com/bookstore/bookstoreapi/data/entity/BookDetails.java

```diff
@@ -0,0 +1,55 @@
+package src.main.java.com.bookstore.bookstoreapi.data.entity;
+
+public class BookDetails {
```

Comment:
As we discussed in the meeting, please edit the `Book.java` entity, this entity class is totally unnecessary.